libs/librrgraph/src/base/rr_graph_storage.h

```diff
@@ -85,8 +88,8 @@ struct alignas(16) t_rr_node_data {
 // t_rr_node_data is a key data structure, so fail at compile time if the
 // structure gets bigger than expected (16 bytes right now). Developers
 // should only expand it after careful consideration and measurement.
-static_assert(sizeof(t_rr_node_data) == 16, "Check t_rr_node_data size");
-static_assert(alignof(t_rr_node_data) == 16, "Check t_rr_node_data size");
+static_assert(sizeof(t_rr_node_data) == 64, "Check t_rr_node_data size");
```

Comment:
Thanks for your comments. I will answer your questions first.

- What will be the changes on the RRSpatialLookup, when introducing the medium node? To be exact, how do you lookup the medium nodes? Through PTC or any other info?

I will find the medium nodes through PTC added in RRSpatialLookup which represents the number of medium mux shown in arch file.

- I see major changes on the rr_node_storage, which introduces additional data. But it may increase the memory footprint significantly. This could be a concern. Therefore, a discussion is required and see how to avoid it.

There will be errors during compilation if the size of t_rr_node_data is set 16. I will try 32 later.

- How is the bend wire modelled? I also see additional data in the rr_graph storage. Are you using a CHANX and CHANY to model a bent wire, and the two nodes are connected through a delayless switch?

We use two nodes connected through a delayless switch to model a bent wire.